android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -186,6 +190,14 @@ fun reset(
     }
 }
 
+fun logOut(
+    onResult: OnResult
+) {
```

Comment:
it doesn't look like we have anything configured in purchases-android.... do you guys have a style config somewhere other than git?

Reply:
we do have it in purchases-android and it should work automatically https://github.com/RevenueCat/purchases-android/tree/develop/.idea/codeStyles but we just recently added this so it's possible it doesn't work as we thought. We should add the same to this repo. 

This indentation is fine though, if using multi line parameters this is the correct indentation. It can also be define as a single line:

```
fun logOut(onResult: OnResult) {
```

And the formatter should not complain. I am going to add to the Kotlin conventions Notion page to do mandatory multi-line for methods with multiple arguments, and optional multiline for single argument methods.